python/pyspark/sql/tests/plot/test_frame_plot_plotly.py

```diff
@@ -0,0 +1,64 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+import pyspark.sql.plot  # noqa: F401
+from pyspark.testing.sqlutils import ReusedSQLTestCase, have_plotly, plotly_requirement_message
+
+
+@unittest.skipIf(not have_plotly, plotly_requirement_message)
```

Comment:
Is this applied correctly? When I tested this PR, this is not skipped like the following.

```
$ python/run-tests.py --python-executables python3 --testnames pyspark.sql.tests.plot.test_frame_plot_plotly
Running PySpark tests. Output is in /Users/dongjoon/APACHE/spark-merge/python/unit-tests.log
Will test against the following Python executables: ['python3']
Will test the following Python tests: ['pyspark.sql.tests.plot.test_frame_plot_plotly']
python3 python_implementation is CPython
python3 version is: Python 3.11.10
Starting test(python3): pyspark.sql.tests.plot.test_frame_plot_plotly (temp output: /Users/dongjoon/APACHE/spark-merge/python/target/c0bba69d-40db-4d01-b9dc-9c5458206143/python3__pyspark.sql.tests.plot.test_frame_plot_plotly__dhnyu9fg.log)
Finished test(python3): pyspark.sql.tests.plot.test_frame_plot_plotly (0s)
Tests passed in 0 seconds

M3-Max ~s:plot_line_w_dep $ pip3 list | grep plotly
sphinx-plotly-directive            0.1.3

[notice] A new release of pip is available: 24.0 -> 24.2
[notice] To update, run: pip install --upgrade pip
```

Reply:
Oh, got it. I randomly chose and it falls into this case. NVM~ Let me test other cases.
- https://github.com/apache/spark/pull/48139#issuecomment-2357825412

Reply:
Does “Tests passed in 0 seconds” indicate that the tests were skipped?